src/System.Text.RegularExpressions/src/System/Text/RegularExpressions/Regex.cs

```diff
@@ -225,6 +222,24 @@ private RegexRunnerFactory Compile(RegexCode code, RegexOptions roptions)
         {
             return RegexCompiler.Compile(code, roptions);
         }
+
+        [SuppressMessage("Microsoft.Naming", "CA1704:IdentifiersShouldBeSpelledCorrectly", MessageId = "assemblyname", Justification = "Microsoft: already shipped since v1 - can't fix without causing a breaking change")]
```

Comment:
Do we run these FxCop naming rules still?  I'm wondering if all of these SuppressMessage attributes can just be removed.

Reply:
I think you are right, we can remove them all.